atmosphere/um_env.yaml

```diff
@@ -0,0 +1,50 @@
+AINITIAL: ''
+ASTART: INPUT/pre-industrial.astart
+AUSCOM_CPL: 'true'
+DATAW: '{work_path}'
+FASTRUN: 'true'
+IDEALISE: ''
+PAREXE: parexe
```

Comment:
I thought this was deleted?

Reply:
Good catch. Due to [this part of the UM code](https://github.com/ACCESS-NRI/UM_v7/blob/dc55b04ac66ee008719c765bc11999d91f1c15cb/umbase_hg3/src/control/top_level/um_shell.F90#L317-L324) the model crashes when the `PAREXE` setting is removed completely from the `um_env.yaml` file. However we can get away with `PAREXE: ''` so that at least it's not pointing to a non existent file. I've just updated this in the config.